uwblueprint/childrens-aid-society
# Juthika/intake create new caregiver modal

## Notion ticket link

[New Caregiver Modal](https://www.notion.so/uwblueprintexecs/Create-New-Caregiver-modal-component-3a52a5650e3c49218f8b1799374eac87)



## Implementation description
* used implementation similar to #104 



## Steps to test
1. checkout branch
2. navigate to individual details page and press the `Add caregivers` button
3. open the modal (right now it might overflow but this should be fixed in #106)
![Screen Shot 2022-12-08 at 1 38 39 PM](https://user-images.githubusercontent.com/77185514/206539368-b78002d9-8903-4856-ab46-782f80bb404b.png)



## What should reviewers focus on?
* right design + behaviours


## Checklist
